quarkus-opentelemetry-exporter-azure/integration-tests/src/test/java/io/quarkiverse/opentelemetry/exporter/it/AzureTest.java

```diff
@@ -50,19 +54,27 @@ void connectionTest() {
                 .statusCode(200)
                 .body("message", equalTo("Direct trace"));
 
-        Awaitility.await()
+        await()
                 .atMost(Duration.ofSeconds(10))
-                .until(azureExportIsDone(wireMockServer));
+                .until(telemetryDataContainTheHttpCall(wireMockServer));
+
+        // Non regression test for https://github.com/Azure/azure-sdk-for-java/issues/41040
+        Thread.sleep(10_000);
```

Comment:
That's a lot of time... You can speed up batch exports by setting [quarkus.otel.bsp.schedule.delay=100](https://quarkus.io/guides/opentelemetry#quarkus-opentelemetry_quarkus-otel-bsp-schedule-delay), or some really tight value for this particular test.
It's probably better to create a new test class for this issue, with a custom [test profile](https://quarkus.io/guides/getting-started-testing#writing-a-profile), that would set that property for you.